packages/runtime/container-runtime/src/opLifecycle/opGroupingManager.ts

```diff
@@ -49,6 +49,37 @@ export class OpGroupingManager {
 		this.logger = createChildLogger({ logger, namespace: "OpGroupingManager" });
 	}
 
+	/**
+	 * Creates a new batch with a single message of type "groupedBatch" and empty contents.
+	 * @param referenceSequenceNumber - reference sequence number
+	 * @returns - IBatch with empty content
+	 */
+	public createEmptyGroupedBatch(
+		resubmittingBatchId: string,
+		referenceSequenceNumber: number,
+	): IBatch<[BatchMessage]> {
+		assert(
+			this.config.groupedBatchingEnabled,
+			"cannot create empty grouped batch when grouped batching is disabled",
+		);
+		const serializedContent = JSON.stringify({
+			type: OpGroupingManager.groupedBatchOp,
+			contents: [],
+		});
+
+		return {
+			contentSizeInBytes: 0,
+			messages: [
+				{
+					metadata: { batchId: resubmittingBatchId, emptyBatch: true },
+					referenceSequenceNumber,
```

Comment:
```suggestion
					metadata: { batchId: resubmittingBatchId, emptyBatch: true },
```

Reply:
Also, put `emptyBatch` on `localOpMetadata` not `metadata`. Doesn't need to be round-tripped, just needs to be accessible in `PSM`.